### PR TITLE
WIP: send tx-tested

### PR DESCRIPTION
**Pull request by ajansari95**

### CODE REVIEW

File: codec/legacy/codec.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package legacy
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
```
Comment (deepanshutr) [inline codec/legacy/codec.go:5]: naming! camel case

File: codec/legacy/codec.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
```
Comment (deepanshutr) [inline codec/legacy/codec.go:6]: naming! camel case

File: codec/legacy/doc.go [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// Package legacy contains a global amino Cdc which is deprecated but
+// still used in several places within the SDK. This package is intended
+// to be removed at some point in the future when the global Cdc is removed.
+package legacy
```
Comment (deepanshutr) [inline codec/legacy/doc.go:4]: retire package if possible

File: codec/types/any.go [hunk 4]
```diff
@@ -0,0 +1,123 @@
+package types
```
Comment (deepanshutr) [inline codec/types/any.go:1]: can this package be imported from somewhere else?

File: codec/unknownproto/benchmarks_test.go [hunk 5]
```diff
@@ -0,0 +1,113 @@
+package unknownproto_test
```
Comment (deepanshutr) [inline codec/unknownproto/benchmarks_test.go:1]: naming!

File: schema/applications/base/simulationApplication.go [hunk 6]
```diff
@@ -17,7 +17,7 @@ import (
 	sdkTypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/auth"
-	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	//"github.com/cosmos/cosmos-sdk/x/auth/exported"
```
Comment (deepanshutr) [inline schema/applications/base/simulationApplication.go:20]: commented code!

File: schema/helpers/message.go [hunk 7]
```diff
@@ -12,6 +12,6 @@ import (
 
 type Message interface {
 	// TODO check if register message code is required
-	RegisterCodec(*codec.Codec)
+	RegisterCodec(codec *codec.LegacyAmino)
```
Comment (deepanshutr) [inline schema/helpers/message.go:15]: named params in interface are redundant

File: codec/legacy/codec.go (hunk 1 above)
Comment (ajansari95) [reply codec/legacy/codec.go:5]: complete codec directory is not needed was using it for some experimental changes would delete it.

**Pull request closed without merging**